specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/examples/ComponentsCreate.json

```diff
@@ -5,7 +5,6 @@
     "resourceGroupName": "my-resource-group",
     "resourceName": "my-component",
     "InsightProperties": {
-      "name": "my-component",
```

Comment:
Removal of model property is a breaking change and requires ARM approval. Requesting ARM review.

Reply:
This is a removal of a read only property from the put example. It just shouldn't be there.